consumer-services/src/main/scala/za/co/absa/spline/consumer/service/repo/ExecutionPlanRepositoryImpl.scala

```diff
@@ -134,6 +134,41 @@ class ExecutionPlanRepositoryImpl @Autowired()(db: ArangoDatabaseAsync) extends
           edges = edges)
     }
   }
+
+  override def getDataSources(value: String, access: Option[String])(implicit ec: ExecutionContext): Future[Array[String]] ={
+    val readResult = db.queryStream[String](
+      """
+        FOR d in dataSource
+        |FOR dp in depends
+        |FILTER dp._from == @planId AND dp._to == d._id
+        |RETURN d.uri
+        |""".stripMargin
+        .stripMargin,
+      Map("planId" -> "executionPlan".concat("/").concat(value)
```

Comment:
We prefer string interpolation to string concatenation.
```suggestion
      Map("planId" -> s"executionPlan/$value")
```

Reply:
added